documentation/modules/configuring/con-config-kafka-connect.adoc

```diff
@@ -43,12 +43,12 @@ spec:
   replicas: 3 # <3>
   # Bootstrap servers (required)
   bootstrapServers: my-cluster-kafka-bootstrap:9092 # <4>
+  groupId: my-connect-cluster # <5>
+  configStorageTopic: my-connect-cluster-configs # <6>
+  offsetStorageTopic: my-connect-cluster-offsets # <7>
+  statusStorageTopic: my-connect-cluster-status # <8>
```

Comment:
We should add the titles like the other sections to align with the other sections.

Also I wonder if we could have all the topics under `6`? This yaml already has 20 bullets underneath and although the topics have different purposes, I expect that users can learn about them in one bullet
```suggestion
  # Connect cluster group ID (required in v1 CRD API)
  groupId: my-connect-cluster # <5>
  # Connect cluster storage topics (required in v1 CRD API)
  configStorageTopic: my-connect-cluster-configs # <6>
  offsetStorageTopic: my-connect-cluster-offsets # <7>
  statusStorageTopic: my-connect-cluster-status # <8>
```